website/docs/guides/visual-editor-qs.md

```diff
@@ -0,0 +1,252 @@
+
+:::
+
+## Access the Visual Editor
+
+To access the Visual Editor:
+
+1. Click **Develop** from the main menu. If you do not see the **Develop** option, ensure you have selected a **Project** from the menu.
+2. Click **Visual Editor**.
+
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/ve-main-menu.png" width="90%" title="Visual Editor in the main menu."/>
+
+3. From the right side, click **Get started** and then click **Create new model**.
+
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/ve-create-new-model.png" width="90%" title="Create a new model from the Visual Editor landing page."/>
+
+## Navigating the interface
+
+The Visual editor comprises a series of menus activated by clicking icons surrounding the border of the larger canvas. With none of the menu items activated, the workspace looks like this:
+
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/ve-screen.png" width="90%" title="The Visual Editor canvas. The number of items is defined in this section." />
+
+Click on an icon to expand its section or execute an action depending on its purpose. The options are as follows:
+
+1. The model's title. This defaults to "Untitled" but can be edited anytime by clicking on it.
+2. The **Operators** toolbar contains the building blocks of creating a model with the editor. 
+3. The **SQL code** area displays the SQL that compiles your model.
+4. The **Runs and previews** that displays run data and previews data for individual operators.
+5. The **Commit history** display.
+6. The **Explorer** view of your model. 
+7. The navigation tab that has icons for (from top to bottom):
+    - Zoom in
+    - Zoom out
+    - Center the model to fit the screen
+    - Auto-layout option for the individual operator tiles
+8. The **Run** command executes `dbt run` for the model.
+9. This button is initially a **Commit** command for your integrated Git provider. It will change to "Open pull request" once your first commit is made. The button will not initially appear until you begin working in the canvas area.
+
+## Create a model
+
+This section will walk you through creating a model with operators using sample data from the [Jaffle Shop](https://github.com/dbt-labs/jaffle-shop) project. With this guide, you will create a basic model that transforms two datasets to build a view of repeat customer purchases while you consider a loyalty program for your shop. 
+
+The operators are the heart of your model. They determine what data will be transformed and how. Click the **+** icon to open the operator menu.
+
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/operators.png" width="90%" title="The operator’s menu on the side of the Visual Editor canvas." />
+
+Read more about the [individual operators](/docs/cloud/visual-editor-interface#operators) to understand the basic purpose of each. Keep in mind that the model you're creating relies on existing models and that the term will primarily be used to reference the model operator in this section.
```

Comment:
I'd clarify the second half of this sentence. Maybe:

"The dbt model created by the Visual Editor builds off of existing models - in these docs, the term "model" will primarily refer to the model operator found this operator drawer."